addons/cppcheckdata.py

```diff
@@ -1621,12 +1621,14 @@ def reportError(location, severity, message, addon, errorId, extra=''):
                 'message': message,
                 'addon': addon,
                 'errorId': errorId,
-                'extra': extra}
+                'extra': misra_severity}
         sys.stdout.write(json.dumps(msg) + '\n')
     else:
         if is_suppressed(location, message, '%s-%s' % (addon, errorId)):
             return
         loc = '[%s:%i]' % (location.file, location.linenr)
+        if len(misra_severity) > 0:
+            message += ' (' + severity + ')'
```

Comment:
this code is strange. is the intention to append the misra severity to the message?